TECH_SPEC.md

```diff
@@ -11,12 +11,12 @@
 - Queue_Manager: smart contract that can be configured by management to hold the optimal withdrawal queues for each vault
 
 # VaultV3 Specification
-The Vault code has been designed as an unopinionated system to distribute funds of depositors into different opportunities (aka Strategies) and manage accounting in a robust way. That's all.
+The Vault code has been designed as an non-opinionated system to distribute funds of depositors into different opportunities (aka Strategies) and manage accounting in a robust way. That's all.
 
-The depositors receive shares of the the vaults token repersentative to their deposit that can then be redeemed or used as yield-bearing tokens.
+The depositors receive shares of the the vaults token representative to their deposit that can then be redeemed or used as yield-bearing tokens.
```

Comment:
"Depositors receive shares (aka vaults tokens) proportional to their deposit amount. Vault tokens are yield-bearing and can be redeemed at any time to get back deposit plus any yield generated."